routes/index.js

```diff
@@ -7,12 +7,11 @@ const constructorMethod = (app) => {
 
     let title = 'Welcome to Unisell';
 
-    if (req.session.user) 
-    {
+    if (req.session.user) {
       title += ', ' + req.session.user.username;
```

Comment:
You can delete this title part i added. I did that for testing

Reply:
I actually like it :)